sdk/core/loader/boot.cc

```diff
@@ -1066,6 +1084,11 @@ extern "C" SchedulerEntryInfo loader_entry_point(const ImgHdr &imgHdr,
 	switcherKey.bounds()      = 1;
 	setSealingKey(imgHdr.scheduler(), Scheduler);
 	setSealingKey(imgHdr.allocator(), Allocator);
+	setSealingKey(imgHdr.object_fast_paths(),
```

Comment:
I still don't really like this name.  Object is very broad.  This is the allocator fast paths library: it holds the allocator's key and adding anything that is not logically part of the allocator to it could break the security.

Reply:
But static ocaps can be used without even having the allocator, once we have something like this PR to mediate unsealing.  In that sense it happens that the allocator holds _the system ocap key_.